test/e2e/wireguard_test.go

```diff
@@ -116,7 +116,7 @@ func testWireGuardTunnelConnectivity(t *testing.T, data *TestData) {
 		var nodeIP string
 		for _, n := range clusterInfo.nodes {
 			if n.name == nodeName1 {
-				nodeIP = n.ip
+				nodeIP = n.ipv4Add
```

Comment:
ditto

Reply:
This test case is in kind cluster.

Reply:
It runs on VMC as well. "test-ipv6-e2e" runs it.